sandbox/grommet-app/src/pages/layouts/kinds/HPEMarketing/MarketingBrochure.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Image,
+  Heading,
+  PageContent,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+import female from '/female.png';
+import clock from '/clock.svg';
+import performance from '/performance.svg';
+
+const ContentArea = ({ alt, image, desc, title }) => {
+  const breakpoint = useContext(ResponsiveContext);
+  console.log(breakpoint);
+  return (
+    <Box align="center" gap="medium">
+      <Box
+        width={breakpoint === 'medium' ? 'xxsmall' : 'xsmall'}
+        height={breakpoint === 'medium' ? 'xxsmall' : 'xsmall'}
+      >
+        <Image fit="contain" alt={alt} src={image} />
+      </Box>
+      <Heading margin="none" level={4}>
```

Comment:
I'm wondering if these should even be headings, but if so they semantically would be level 2
```suggestion
      <Heading margin="none" level={2}>
```

Reply:
I just changed to `Text`